whois-update/src/main/java/net/ripe/db/whois/update/handler/validator/route/OriginValidator.java

```diff
@@ -69,7 +69,7 @@ public void validate(PreparedUpdate update, UpdateContext updateContext) {
         }
     }
 
-    private boolean isReservedAsNumber(Long asn) {
+    public boolean isReservedAsNumber(Long asn) {
```

Comment:
Better not to expose private methods inside validators, but to move the logic out to a separate class.

Can you create something like ReservedAutnum in the same package as net.ripe.db.whois.update.domain.LegacyAutnum? Then move isReservedAsNumber to it.